x-pack/auditbeat/module/system/package/package_homebrew_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build !windows
+
+package pkg
+
+import (
+	"os"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/elastic/beats/auditbeat/core"
+	abtest "github.com/elastic/beats/auditbeat/testing"
+	"github.com/elastic/beats/libbeat/beat"
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+)
+
+func TestHomebrew(t *testing.T) {
+	defer abtest.SetupDataDir(t)()
+
+	oldPath := homebrewCellarPath
+	defer func() {
+		homebrewCellarPath = oldPath
+	}()
+	homebrewCellarPath = "../../../tests/files/homebrew/"
+
+	// Test just listBrewPackages()
+	packages, err := listBrewPackages()
+	assert.NoError(t, err)
+	assert.Len(t, packages, 1)
+	pkg := packages[0]
```

Comment:
This will panic if there’s a failure in the above assertion. Probably should go with `if !assert.Len() { return }` or similar.

Reply:
Thanks, I've guarded the access with an `if`.